### DISCUSSION TITLE
appropriate tool for driven overdamped rigid bodies

### DISCUSSION THREAD

**Discussion by noambernstein** [glotzerlab/hoomd-blue#1976] [General]
I'm interested in simulating a 2D collection of rectangular solids, purely repulsive, that are being dragged around a frictional surface, i.e. one or more particles are forced to travel along a specified path (at a specified speed, but I'm aiming for overdamped dynamics, so I'm not sure that matters) and the others are pushed around by this forcing.

Does this sound like something I could do with hoomd-blue (or, if not, does anyone know of another tool that'd work)? I considered both hard particle MC and rigid body overdamped-Langevin MD, but in both cases it wasn't obvious to me from the documentation and examples I found whether it was feasible, or even in principle possible, to apply the forcing I want.

**Comment by joaander** [comment]
Exclude the forced particles from integration and apply the prescribed motion with a `CustomUpdater`: https://hoomd-blue.readthedocs.io/en/v5.0.0/tutorial/04-Custom-Actions-In-Python/00-index.html
